main.js

```diff
@@ -0,0 +1,139 @@
+const close = document.querySelector('.close');
+function toggleMenu() {
+  const menu = document.querySelector('#menu');
+  menu.style.display = menu.style.display === 'block' ? 'none' : 'block';
+}
+burger.addEventListener('click', toggleMenu);
+close.addEventListener('click', toggleMenu);
+
+// Generate speakers section
+
+const speakers = [
+  {
+    name: 'Dario Scilipoti',
+    title: 'CEO | Bluebird Coffee Roastery',
+    img: 'assets/speakers/dario.png',
+    bio: 'Dario is the CEO of Bluebird Coffee Roastery, a specialty coffee roastery based in KZN',
+    catergory: 'coffee',
+  },
+  {
+    name: 'Ichsan Sandypratama',
+    title: 'CEO | Google',
+    img: 'assets/speakers/ichsan.jpg',
+    bio: 'Ichsan is the CEO of Google, a search engine based in the US. He really likes coffee.',
+    catergory: 'code',
+  },
+  {
+    name: 'Manzi Tresor',
+    title: 'Chief Design Officer | Apple',
+    img: 'assets/speakers/manzi.jpeg',
+    bio: 'As the Chief Design Officer of Apple, Manzi is the visionary driving the company\'s aesthetic and user experience strategy.',
+    catergory: 'code',
+  },
+  {
+    name: 'James Hoffmann',
+    title: 'Weird Coffee Guy | YouTube',
+    img: 'assets/speakers/james.jpeg',
+    bio: 'A world-renowned coffee expert and champion, is a captivating figure in the specialty coffee industry, inspiring coffee enthusiasts with his knowledge, passion, and engaging storytelling.',
+    catergory: 'coffee',
+  },
+  {
+    name: 'Winston Thomas',
+    title: 'South African Barista Champion 2020',
+    img: 'assets/speakers/winston.png',
+    bio: 'The South African Barista Champion is a master of his craft, capturing the hearts and palates of coffee enthusiasts with his exceptional skills and creativity.',
+    catergory: 'coffee',
+  },
+  {
+    name: 'Alex Lee',
+    title: 'Java Programming Educator | YouTube',
+    img: 'assets/speakers/alex.png',
+    bio: 'Alex Lee is a charismatic and knowledgeable instructor who simplifies complex programming concepts, making Java accessible and enjoyable for learners of all levels.',
+    catergory: 'code',
+  },
+];
+
+function genSpeakers(loopTo) {
+  const title = document.createElement('h3');
+  title.textContent = 'Featured Speakers';
+  const hr = document.createElement('hr');
+  const speakersSection = document.querySelector('#speakers');
+  speakersSection.appendChild(title);
+  speakersSection.appendChild(hr);
+  const container = document.createElement('div');
+  for (let i = 0; i < loopTo; i += 1) {
+    container.classList.add('speaker-content');
+    const speakerCard = document.createElement('div');
+    speakerCard.classList.add('speaker');
+    speakerCard.innerHTML = `
+      <img src="${speakers[i].img}" alt="${speakers[i].name}">
+      <section>
+        <h3>${speakers[i].name}</h3>
+        <p class="subheading">${speakers[i].title}</p>
+        <hr>
+        <article>${speakers[i].bio}</article>
+      </section>
```

Comment:
- [ ] Good implementation of the speaker section using JavaScript to perform this action 👏 , There is some point you still need to improve on the design. You will notice with me that the part in rounded in red on picture below should stick to the Figma design. You use two kind in your web page. Kindly use the same thing as in the given design 👍

| **Your design** |  **Cindy Design** |
| :---: | :---: |
| ![Capture1](https://github.com/aamir-asaram/capstone-one/assets/80458199/3666a9cc-4b07-40c4-86e3-160c3025a383) | ![Capture](https://github.com/aamir-asaram/capstone-one/assets/80458199/8736c8fa-969a-400b-863a-56beb33d69b0) |

Reply:
Hi @iambenkis, I appreciate the review! The use of two kinds of background was intentional as the coffee beans represent a coffee-related speaker and the binary represents a code-related speaker [I mentioned this in the PR description]. As per the design guideline I was under the impression we could change pictures slightly so is it okay if I keep this change? Or is it necessary that I change it back? Thanks in advance :smile: